src/audio/code_8019AF00.c

```diff
@@ -5030,7 +5324,16 @@ void Audio_UpdateEnemyBgmVolume(f32 dist) {
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_801A3AC0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_801A3AEC.s")
+void Audio_PlaySfx_Window(u8 windowToggleDirection) {
```

Comment:
What kind of window? I assume this is a GUI element not like.. an ingame window?

Reply:
It's specifically for opening and closing the pause menu. You're right, `PauseMenu` could be used instead of `Window`

Reply:
I think the name `Window` came from the original sfx name `NA_SE_SY_WIN_OPEN` and `NA_SE_SY_WIN_CLOSE`